pkg/synopsysctl/cmd_start.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright (C) 2019 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package synopsysctl
+
+import (
+	"fmt"
+
+	"github.com/blackducksoftware/synopsys-operator/pkg/util"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// startCmd starts a resource in the cluster if it's stopped
+var startCmd = &cobra.Command{
+	Use:   "start [resource]",
+	Short: "Start a Synopsys Resource in your cluster",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return fmt.Errorf("Must specify a resource")
+	},
+}
+
+// startBlackduckCmd starts a Blackduck in the cluster
+var startBlackduckCmd = &cobra.Command{
```

Comment:
@yashbhutwala 
Will the start and stop for opssight and alert will be in separate PR?

Reply:
Done.